pages/ksphere/d2kf/1.0.1-0.3.0/user-management/index.md

```diff
@@ -0,0 +1,140 @@
+
+## Assigning Permissions to Kubeflow Users
+
+_NB: The following tutorial assumes you have already connected an OIDC provider to your Konvoy cluster via Konvoy's built-in Dex integration. If you have not yet done so, review the documentation [here](https://docs.d2iq.com/ksphere/konvoy/latest/security/external-idps/) before proceeding._
+
+### Adding Permissions for a Kubeflow Administrator     
+
+#### Step 1 - Create the `ClusterRoleBinding` YAML manifest
+
+In your terminal shell, run:
+
+```bash
+cat << EOF > add-kubeflow-admin.yaml                                        
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: "<name of user>"
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: kubeflow-admin
+subjects:
+- apiGroup: rbac.authorization.k8s.io
+  kind: User
+  name: "<Email of user>"
+EOF
+
+```
+
+#### Step 2 - Edit `add-kubeflow-admin.yaml` with details for the new user
+
+Edit the `metadata.name` to the name of the user you want to add. 
+
+**Note** This name should be globally unique across your Konvoy cluster, otherwise you may overwrite another user's permissions.
+
+Edit the `subjects.apiGroup.name` to match the email address associated with the user's OIDC account.
+
+
+#### Step 3 - Apply `add-kubeflow-admin.yaml` to your Konvoy cluster
+
+In your terminal shell, run:
+
+`kubectl apply -f add-kubeflow-admin.yaml` 
+
+Your output should look similar to this:
+
+```bash
+kubectl apply -f add-kubeflow-admin.yaml                              
+
+clusterrolebinding.rbac.authorization.k8s.io/<name of user> created
+```
+
+### Adding Permissions for a Kubeflow User
+
+#### Step 1 - Create the `ClusterRoleBinding` YAML manifest
+
+In your terminal shell, run:
+
+```bash
+cat << EOF > add-kubeflow-user.yaml                                        
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: "<name of user>"
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: kubeflow-edit
+subjects:
+- apiGroup: rbac.authorization.k8s.io
+  kind: User
+  name: "<Email of user>"
+EOF
+
+```
+
+#### Step 2 - Edit `add-kubeflow-user.yaml` with details for the new user
+
+Edit the `metadata.name` to the name of the user you want to add. 
+
+**Note** This name should be globally unique across your Konvoy cluster, otherwise you may overwrite another user's permissions.
+
+Edit `subjects.apiGroup.name` to match the email address associated with the user's OIDC account.
+
+
+#### Step 3 - Apply `add-kubeflow-user.yaml` to your Konvoy cluster
+
+In your terminal shell, run: 
+
+`kubectl apply -f add-kubeflow-user.yaml` 
+
+Your output should look similar to this:
+
+```bash
+kubectl apply -f add-kubeflow-user.yaml                              
+
+clusterrolebinding.rbac.authorization.k8s.io/<name of user> created
+```
+
+# Permissions Charts
```

Comment:
these charts look odd to me. @akirillov - can you look over them and make sure that k9s is outputting the permissions correctly here?